about.html

```diff
@@ -0,0 +1,151 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
+    crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap">
+    <title>About-Capstone</title>
+</head>
+<body>
+   <header class="header">
+       <nav class="nav">
+            <div class="icon" id="hamburger">
+               <i class="fas fa-bars"></i>
+            </div>
+            <div class="logo">
+                <a href="index.html">
+                    <i class="fab fa-bitcoin">Bitcoin</i>
+                </a>
+            </div>
+            <ul class="top-nav">
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class="fab fa-facebook-f"></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">English</a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">Logout</a>
+                </li>
+            </ul>
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <div class="icon">
+                       <i class="fas fa-times" id="close-icon"></i>
+                    </div>
+                </li>
+              <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+              <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+            </ul>
+        </nav>
+        <div class="hero-container">
+            <div class="hero-about-content">
+                <h4 class="hero-about-content__heading">
+                   BLOCK CHAIN SUMMIT 2021
+                </h4>
+                <p class="hero-about-content__desc bg-white">
+                 The Block Chain Summit brings together community of
+                 experts, academics, and activists who comprise the Creative
+                 Commons affliate network in a different country
+                 every two years. This year, we hope to expand our invitation list
+                 including organizations and individuals
+                </p>
+                <div class="hero-about-content-contact">
+                    <p>
+                       Please contact us per Email for any
+                       further questions about Block Chain Summit
+                       2021!
+                    </p>
```

Comment:
- [ ] As the previous reviewer stated, there should be one but not more than one `h1` tag on each page. The `about.html` page currently does not have an `h1` tag